src/azle.ts

```diff
@@ -34,23 +34,42 @@ const GLOBAL_AZLE_TARGET_DIR = `${GLOBAL_AZLE_CONFIG_DIR}/target`;
 azle();
 
 function azle() {
+    const install_rustup_path = path.resolve(__dirname, '../install_rustup.sh');
+
+    // TODO research exec vs spawn
+    execSync(`${install_rustup_path} ${version} ${rust_version}`, {
+        stdio: 'inherit'
+    });
+
     const stdioType = getStdIoType();
     const canisterName = unwrap(getCanisterName(process.argv));
+    const canister_path = `.azle/${canisterName}`;
```

Comment:
This is stylistic so probably not worth changing, but you're mixing camel and snake case. I think this file is written all in camelCase up till now.